src/tests/dotnet-counters/DotnetCounters.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Don't we want both here?

Reply:
counters and trace weren't very interesting to test in multiple runtimes.